requirements-py3.txt

```diff
@@ -15,7 +15,6 @@ mccabe==0.4.0
 MechanicalSoup==0.11.0
 mock==2.0.0
 more-itertools==5.0.0
-mysqlclient==1.3.9
```

Comment:
Please add mysqlclient.

Reply:
ok. 
Just wanted to inform you that
mysqlclient==1.3.13 is present in requirements-common.txt.

Reply:
Ok. then please ignore this.